### DISCUSSION TITLE
How to config HA-Proxy Tunnel in order to use it as a Relay server on Hiddify? | چگونه می‌توان سرور ریلی با روش HA-Proxy ایجاد کرد و از آن در هیدیفای استفاده نمود؟

### DISCUSSION THREAD

**Discussion by lymanjre** [hiddify/Hiddify-Manager#3443] [Q&A سوالات رایج] [answered]
How to config HA-Proxy Tunnel in order to use it as a Relay server on Hiddify? 

 چگونه می‌توان سرور ریلی با روش HA-Proxy ایجاد کرد و از آن در هیدیفای استفاده نمود؟

**Comment by ReturnFI** [comment, marked as answer]
# جهت مشاهده مقاله آپدیت شده در خصوص این روش [اینجا](https://hiddify.com/fa/manager/domain-worker-cdn-and-tunneling/Tutorial-for-setting-up-HA-Proxy-tunnel-as-relay-server/) را ببینید.

**Reply by darkness3676** [reply]
درود و وقت بخیر
امکانش هست توضیح بدید چطور ادرس پنل رو هم تانل کنیم ؟ به عبارتی از طریق تانل وارد پنل بشیم ، مثل پنل سنایی ، یا پنل های غیر هیدیفای ، برای تولید لینک ها
ظاهرا فقط روی Mode tcp تانل میشه
